### PR TITLE
Increase test coverage of query-service

### LINKED ISSUE #1737: Write unit tests for query-service.tsx

### PR DESCRIPTION
**Pull request by bamader**
# Query Service Test Coverage

## Summary
This PR adds unit and e2e tests into our app to complete coverage of the query service. Helper and data manipulation functions used by the query-service are tested in jest's unit framework, while tests of the larger, central `generalizedQuery` function are better suited to integration to verify that patient results are found.

## Related Issue
Fixes #1737

### CODE REVIEW

File: containers/tefca-viewer/src/app/tests/shared_utils/readJsonFile.ts
Comment (m-goggins) [inline containers/tefca-viewer/src/app/tests/shared_utils/readJsonFile.ts:1]: Should this be it's own file or should it be something like `utils.ts`? I don't feel too strongly though

File: containers/tefca-viewer/src/app/tests/unit/format-service.test.tsx [hunk 1]
```diff
@@ -465,3 +466,29 @@ describe("FormatPhoneAsDigits", () => {
     expect(FormatPhoneAsDigits(givenPhone)).toEqual(expectedResult);
   });
 });
+
+describe("GetPhoneQueryFormats", () => {
+  it("should fail gracefully on partial phone number inputs", async () => {
```
Comment (m-goggins) [inline containers/tefca-viewer/src/app/tests/unit/format-service.test.tsx:471]: What happens if an empty string is passed in?

**Review by m-goggins — COMMENTED** [review]
Thanks for adding all of these tests! I was wondering what would happen if you pass in an empty string to phone number (or phone number) is not available?

**Review by m-goggins — APPROVED** [review]

**Review by robertandremitchell — COMMENTED** [review]
been unable to get playwright to run. @bamader , would you have a little time today or tomorrow to maybe help debug?

**Review by robertandremitchell — APPROVED** [review]
looks good! thanks for helping debug!
